packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-node.vue

```diff
@@ -17,6 +17,7 @@ import { WorkflowNode, WorkflowPortStatus } from '@/types/workflow';
 import Button from 'primevue/button';
 import TeraOperatorPlaceholder from '@/components/operator/tera-operator-placeholder.vue';
 import { cloneDeep } from 'lodash';
+import { blankIntervention } from '@/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation';
```

Comment:
I think this `blankIntervention` should be in the _Intervention Policy_ operation file.